Haskell/Haskell.sublime-syntax

```diff
@@ -249,10 +249,11 @@ contexts:
   preprocessor:
     # In addition to Haskell's "native" syntax,
     # GHC permits the C preprocessor to be run on a source file.
-    - match: ^\s*(#)\s*\w+
+    - match: ^\s*(#)\s*(\w+)
       scope: meta.preprocessor.c
       captures:
         1: punctuation.definition.preprocessor.c
+        2: keyword.directive.other.c
```

Comment:
Can you change it to

```yaml
- match: (^\s*#\s*\w+)
  scope: meta.preprocessor.c
  captures:
    1: keyword.directive.other.c
- include: pragma
```

it would look much better imo

Reply:
Included the `#` and maintained punctuation as this is the way we are asked to scope such things.

Reply:
@SlavKing98 Monokai doesn't scope `punctuation.definition`. Try a more modern color scheme like Mariana or Breakers.